packages/icons/lib/icons.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path')
+const dist = [__dirname, '..', 'dist']
```

Comment:
Can we use TS for new packages?

You can just do this: https://github.com/cypress-io/cypress/blob/10.0-release/scripts/gulp/gulpfile.ts#L243

- `yarn gulp makePackage` And it will add a package with TS etc etc for you.